src/main/resources/test_files/runtime/InstanceOf2.jq

```diff
@@ -0,0 +1,4 @@
+(:JIQS: ShouldRun; Output="true":)
+() instance of string?
+
```

Comment:
I assume that you separated these tests because of () on the left hand side. If you fix the instance of iterator to return a false boolean item in next(), this separation will no longer be needed as instance of will correctly return false/true even even if the instance of expression is nested in the query.

Reply:
Correct. 
Now, with the fixed version of InstanceOfIterator, all the tests are in a single file.